src/containers/find-offer/offer-container/OfferContainer.tsx

```diff
@@ -2,7 +2,8 @@ import { FC } from 'react'
 import Box from '@mui/material/Box'
 import { Grid } from '@mui/material'
```

Comment:
Import like this:
import Grid from '@mui/material/Grid '

Reply:
Fixed.